Compteur/compteur/src/Counters.js

```diff
@@ -0,0 +1,11 @@
+import Counter from './Counter';
+
+const Counters = ({counters, onDelete}) => {
+  return (
+    <>
+      {counters.map((counter) => (<Counter key={counter.id} a_counter={counter} onDelete={onDelete} />))}
```

Comment:
très bien, tu as bien saisi l'exercice.

Dernière chose, là c'est pas grave mais sur tes futures itérations sur un tableau ou un objet, il faut avoir une key plus spécifique par convention.

**convention:**

car dans un vrai projet, tu pourrais être amené à avoir plusieurs listes à afficher, pour éviter un conflit dans les keys, et pour que React intéragisse correctement sur ses render, il faut avoir des keys spécifiques.

ici par exemple 
```
key={`counters-list-${counter.id}`}
``` 
ferait l'affaire et au moins, pas de mauvaise surprise